tests.cmake

```diff
@@ -77,13 +77,32 @@ MACRO(BUILD_TEST FILE_NAME)
   # Link against Boost.
   TARGET_LINK_LIBRARIES(${EXE_NAME}${PROGRAM_SUFFIX} ${Boost_LIBRARIES})
 
+  if(WIN32)
+    set(SEP "\\;")
+  else()
+    set(SEP ":")
+  endif()
+
   # Make sure the plugins will be found.
   SET_PROPERTY(
     TEST ${EXE_NAME}${PROGRAM_SUFFIX} PROPERTY ENVIRONMENT
-    "LTDL_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src:$ENV{LTDL_LIBRARY_PATH}")
+    "PATH=${Boost_LIBRARY_DIR_RELEASE}${SEP}${CMAKE_INSTALL_PREFIX}/bin${SEP}${CMAKE_INSTALL_PREFIX}/lib/roboptim-core${SEP}$ENV{PATH}"
```

Comment:
Similarly, LD_LIBRARY is unix only

Le mar. 9 août 2016 21:49, Keith François <francois.keith@gmail.com> a
écrit :

> Right. i can update the PR accordingly
>
> Le mar. 9 août 2016 21:34, Benjamin Chrétien <notifications@github.com> a
> écrit :
>
>> In tests.cmake
>> <https://github.com/roboptim/roboptim-shared-tests/pull/24#discussion_r74129110>
>> :
>>
>> >    # Make sure the plugins will be found.
>> >    SET_PROPERTY(
>> >      TEST ${EXE_NAME}${PROGRAM_SUFFIX} PROPERTY ENVIRONMENT
>> > -    "LTDL_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src:$ENV{LTDL_LIBRARY_PATH}")
>> > +    "PATH=${Boost_LIBRARY_DIR_RELEASE}${SEP}${CMAKE_INSTALL_PREFIX}/bin${SEP}${CMAKE_INSTALL_PREFIX}/lib/roboptim-core${SEP}$ENV{PATH}"
>>
>> Updating PATH is only required for Windows (MSVC?) to find libraries,
>> right?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/roboptim/roboptim-shared-tests/pull/24/files/5e439d1f9aa9c9e9de49dd87fc8895aa41c1ac80#r74129110>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AAaYEm2mXJLemm6X7_Iku4ZlV2LoAz2uks5qeNYxgaJpZM4Jflfi>
>> .
>>
>